benchmark/common.js

```diff
@@ -238,20 +238,18 @@ Benchmark.prototype.report = function(value) {
     console.log('%s: %s', heading, value.toFixed(5));
   else if (outputFormat == 'csv')
     console.log('%s,%s', heading, value.toFixed(5));
-
-  process.exit(0);
 };
 
 Benchmark.prototype.getHeading = function() {
   var conf = this.config;
 
   if (outputFormat == 'default') {
     return this._name + ' ' + Object.keys(conf).map(function(key) {
-      return key + '=' + conf[key];
+      return key + '=' + JSON.stringify('' + conf[key]);
     }).join(' ');
   } else if (outputFormat == 'csv') {
     return this._name + ',' + Object.keys(conf).map(function(key) {
-      return conf[key];
+      return JSON.stringify('' + conf[key]);
     }).join(',');
   }
 };
```

Comment:
Initially I had planned it to be in a separate commit but there would then technically be a gap where the new benchmark would be broken (causing the benchmark runner to get in a large loop actually). I can amend the commit message...